types/node-telegram-bot-api/index.d.ts

```diff
@@ -148,6 +148,7 @@ declare namespace TelegramBot {
         certificate?: string | Stream | undefined;
         max_connections?: number | undefined;
         allowed_updates?: string[] | undefined;
+        secret_token?: string;
```

Comment:
Please consider changing the type to string | undefined so that the behavior is consistent to other options, i.e. max_connections.